bootstrap.py

```diff
@@ -179,43 +177,9 @@ def remove_pyls_installation():
 
 atexit.register(remove_pyls_installation)
 
-print("*. Declaring completion providers")
-# Register completion providers
-fallback = pkg_resources.EntryPoint.parse(
-    'fallback = spyder.plugins.completion.providers.fallback.provider:'
-    'FallbackProvider')
-
-snippets = pkg_resources.EntryPoint.parse(
-    'snippets = spyder.plugins.completion.providers.snippets.provider:'
-    'SnippetsProvider'
-)
-
-lsp = pkg_resources.EntryPoint.parse(
-    'lsp = spyder.plugins.completion.providers.languageserver.provider:'
-    'LanguageServerProvider'
-)
-
-kite = pkg_resources.EntryPoint.parse(
-    'kite = spyder.plugins.completion.providers.kite.provider:'
-    'KiteProvider'
-)
-
-# Create a fake Spyder distribution
-d = pkg_resources.Distribution(__file__)
-
-# Add the providers to the fake EntryPoint
-d._ep_map = {
-    'spyder.completions': {
-        'fallback': fallback,
-        'snippets': snippets,
-        'lsp': lsp,
-        'kite': kite
-    }
-}
-
-# Add the fake distribution to the global working_set
-pkg_resources.working_set.add(d, 'spyder')
-
+# Create Spyder egg info
+subprocess.check_output([sys.executable, '-W', 'ignore', 'setup.py',
+                         'develop', '--no-deps'])
```

Comment:
This totally defeats the purpose of our bootstrap script, which is not to alter people's envs at all. So please revert this change.

Reply:
done